plugins/bazaar/src/api.ts

```diff
@@ -81,60 +106,49 @@ export class BazaarClient implements BazaarApi {
     return response.ok ? response : null;
   }
 
-  async getMembers(entity: Entity): Promise<any> {
+  async getMembers(id: number): Promise<any> {
     const baseUrl = await this.discoveryApi.getBaseUrl('bazaar');
 
-    return await fetch(
-      `${baseUrl}/members/${encodeURIComponent(stringifyEntityRef(entity))}`,
-      {
-        method: 'GET',
-      },
-    ).then(resp => resp.json());
+    return await fetch(`${baseUrl}/projects/${id}/members`, {
+      method: 'GET',
+    }).then(resp => resp.json());
   }
 
-  async addMember(entity: Entity): Promise<void> {
+  async addMember(id: number, userId: string): Promise<void> {
     const baseUrl = await this.discoveryApi.getBaseUrl('bazaar');
 
-    await fetch(`${baseUrl}/member`, {
+    await fetch(`${baseUrl}/projects/${id}/member/${userId}`, {
```

Comment:
remember to url encode strings such as the user ID, always (here and some other places in this file)